code/Language/Drasil/Chunk/Attribute/ShortName.hs

```diff
@@ -0,0 +1,6 @@
+{-# Language TemplateHaskell #-}
+module Language.Drasil.Chunk.Attribute.ShortName where
+
+import Language.Drasil.Spec (Sentence(..))
+
+data ShortNm = ShortName Sentence
```

Comment:
Shouldn't this also be changed for consistency?

Reply:
Yes, updated in commit 02bbeb148.